### ISSUE TITLE
sizing_mode="stretch_width" does not work

### ISSUE THREAD

**Issue by MarcSkovMadsen** [jmosbacher/pydantic-panel#15]
I almost always use `sizing_mode="stretch_width"`. But I can see its not supported.

![image](https://user-images.githubusercontent.com/42288570/182012799-6dde256d-2601-4758-b684-ce77869c6e76.png)

```python
import pydantic


class SomeModel(pydantic.BaseModel):
   name: str
   value: float

import panel as pn

import pydantic_panel as pp

pn.extension()

pn.panel(SomeModel, sizing_mode="stretch_width")

widget = pn.panel(SomeModel)
layout = pn.Column(widget, widget.json)
layout.servable()
```

**Comment by MarcSkovMadsen** [comment]
## Workaround

Adding `pn.extension(sizing_mode="stretch_width")` makes it work

![image](https://user-images.githubusercontent.com/42288570/182012852-0590a83c-33d3-477d-9006-49722b66888c.png)

**Comment by MarcSkovMadsen** [comment]
Please note I added a request to get the margin of the `JSON` pane adjusted. See https://github.com/holoviz/panel/issues/3736

**Comment by jmosbacher** [comment]
Yeah I still havent decided on a strategy for propagating sizing options down to sub widgets. 
Some sub widgets are naturally larger then others and also depends on whether the subwidgets are in a Row-like container or Column-like. SO I'm expecting it to be not trivial to do right.
Suggestions are very welcome, although probably wont have time to implement in very near future.
